sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_system_events_to_custom_topic.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import os
+import json
+from random import randint, sample
+from typing import Sequence
+import time
+
+PACKAGE_PARENT = '..'
+SCRIPT_DIR = os.path.dirname(os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
+sys.path.append(os.path.normpath(os.path.join(SCRIPT_DIR, PACKAGE_PARENT)))
+
+from azure.core.credentials import AzureKeyCredential
+from azure.mgmt.eventgrid import EventGridManagementClient
+from azure.eventgrid import EventGridPublisherClient
+from azure.eventgrid import EventGridEvent, CloudEvent
+#from azure.eventgrid._generated.event_grid_publisher_client.models._models import StorageBlobCreatedEventData
+from azure.core.exceptions import (
+    ResourceNotFoundError,
+    ResourceExistsError,
+    ClientAuthenticationError
+)
+
+import logging
+
+#logging.basicConfig(level=logging.DEBUG)
+
+key = os.environ.get("CUSTOM_ACCESS_KEY")
+topic_hostname = "eventgridcloudeventsub.eastus-1.eventgrid.azure.net"
+
+# authenticate client
+credential = AzureKeyCredential(key)
+client = EventGridPublisherClient(topic_hostname, credential)#, logging_enable=True)
+
+#data_obj = StorageBlobCreatedEventData(
```

Comment:
deprecated?

Reply:
yes, going to remove this entire file from commit. This was only for testing consumption of system event samples.